Classes/YLTableView.m

```diff
@@ -71,7 +73,7 @@ - (void)registerClass:(Class)headerFooterViewClass forHeaderFooterViewReuseIdent
   }
 }
 
-- (YLTableViewCell *)sizingCellForReuseIdentifier:(NSString *)reuseIdentifier {
+- (nonnull YLTableViewCell *)sizingCellForReuseIdentifier:(nonnull NSString *)reuseIdentifier {
   NSAssert(reuseIdentifier != nil, @"Must have a reuse identifier.");
```

Comment:
I think the assert is still valuable; there's plenty of nonaudited code calling into this, and there are ways to still pass nil indirectly.